AgentInterfaces/SessionHostHeartbeatInfo.cs

```diff
@@ -59,6 +59,11 @@ public class SessionHostHeartbeatInfo
         /// </summary>
         public DateTime? NextScheduledMaintenanceUtc { get; set; }
 
+        /// <summary>
+        /// Schedule of the next planned maintenance
+        /// </summary>
+        public MaintenanceSchedule NextScheduledMaintenance { get; set; }
```

Comment:
The name "NextScheduledMaintenance" made more sense when it referred to the next upcoming maintenance event--I think the entire schedule should be called "ScheduledMaintenance" or "MaintenanceSchedule"

Reply:
assuming that we'll do the work of informing the user about all maintenance events (e.g. one generated from PlayFab and one from Azure - if they co-exist), I'm fine with this. Will amend.